exercises/collatz-conjecture/examples/src/CollatzConjecture.purs

```diff
@@ -0,0 +1,15 @@
+module CollatzConjecture
+  ( collatz
+  ) where
+
+import Prelude
+import Data.Int (even)
+import Data.Maybe (Maybe(..))
+
+collatzHelper :: Int -> Int -> Int
```

Comment:
Should this function have a `Partial` constraint? 

😆

Reply:
I have an `otherwise` guard so `Partial` doesn't seem necessary. The partiality is captured in the main `collatz` function using `Maybe`

Reply:
Ah, my brain was in PR review mode rather than Fields medal mode